src/lib/bootloader/grub2base.rb

```diff
@@ -56,7 +56,7 @@ def pmbr_setup(action, *devices)
     # Propose global options of bootloader
     def StandardGlobals
       # s390 do not have os_prober, see bnc#868909#c2
-      disable_os_prober = Arch.s390 || ProductFeatures.GetBooleanFeature("globals", "disable_os_prober")
+      disable_os_prober = (Arch.s390||Arch.ppc) || ProductFeatures.GetBooleanFeature("globals", "disable_os_prober")
```

Comment:
I do not see oficial support for it. If it is goal to have support for ppc in opensuse I am fine with this change. just please add reference to ppc bug in comment above similar like I did for s390, so in future we know why it is there and have reference.